java/src/main/native/src/ColumnVectorJni.cpp

```diff
@@ -726,6 +727,23 @@ JNIEXPORT jlong JNICALL Java_ai_rapids_cudf_ColumnVector_stringConcatenation(JNI
   CATCH_STD(env, 0);
 }
 
+JNIEXPORT jlong JNICALL Java_ai_rapids_cudf_ColumnVector_interleaveColumns(JNIEnv *env, jobject j_object,
+                                                                  jlongArray column_handles) {
```

Comment:
Nit: Not sure what indentation rule this is trying to follow, seems arbitrary.

Reply:
I followed the same style (or lack of) as the surrounding methods. I agree that it seems arbitrary.

Reply:
Resolved with latest changes.